src/plone/app/multilingual/browser/setup.py

```diff
@@ -266,7 +266,7 @@ def setupLanguageSwitcher(self):
 
         if 'language-switcher' not in site.view_methods:
             methods = site.view_methods
-            site.view_methods = methods + ('language-switcher', )
+            site.view_methods = list(methods) + ['language-switcher', ]
```

Comment:
After re-saving the FTI under ZMI -> portal_types for `Plone Site` the `site.view_methods` seems to be `tuple`. Not sure if this is a problem, but it seems related to #337.

Reply:
Thanks, that might be the cause!